Source/GNUmakefile

```diff
@@ -584,6 +584,7 @@ GSWindowDecorationView.h \
 GSXibElement.h \
 GSXibLoading.h \
 GSXibKeyedUnarchiver.h \
+GSXib5KeyedUnarchiver.h \
```

Comment:
I actually think we should integrate Xib5 into Xib since there really is no need for the other one.

> You did see that I already move the toolbar over? As for the rest of the values in this and the other classes I didn't touch them as I am quite sure that you simple change is wrong. I would have preferred to come up with a correct solution here not just something that basically works but may have side effects when loading old XIB files.

I'm not done yet. :)